### DISCUSSION TITLE
QiitaCLI投稿時のためにタグを検索する機能が欲しい

### DISCUSSION THREAD

**Discussion by TheJuniorTheSenior** [increments/qiita-discussions#524] [要望・アイデア・不具合]
普段、Qiitaで記事を投稿するとき、タグ入力バーに文字を入力したら自動的にタグを検索し、候補とその記事数を表示してくれるようになっていますね。

これは、タグの名寄せの労力を削減することに貢献している機能だと思います。

しかし、Qiita CLIを使って投稿するとき、この機能は使えませんよね。

代わりに、タグを検索できるページか何かを作ってほしいです。

投稿する側も、よく使われているタグを探す手間が省けます。

いかがでしょうか。

**Comment by tanaka-asami415** [comment]
@TheJuniorTheSenior 
ご意見ありがとうございます！

直近の開発優先度などから、すぐの対応はできないですが
何か方法がないか検討させていただきます🙇
